samples/ImageList/src/views/ImageList.tsx

```diff
@@ -106,7 +106,8 @@ export class ImageList extends ComponentBase<{}, ImageListState> {
         };
     }
 
-    private _renderItem = (item: ImageListItemInfo) => {
+    private _renderItem = (details: VirtualListViewCellRenderDetails<ImageListItemInfo>) => {
```

Comment:
@berickson1 I think we can use, instead of an additional variable. What do you think?

```typescript
private _renderItem = ({ item }: VirtualListViewCellRenderDetails<ImageListItemInfo>) => {

```